packages/core/features/src/connect.ts

```diff
@@ -65,12 +65,24 @@ export type ConnectMethod = StandardConnectMethod;
  */
 export interface StandardConnectInput {
     /**
-     * By default, using the {@link StandardConnectFeature} should prompt the user to request authorization to accounts.
-     * Set the `silent` flag to `true` to request accounts that have already been authorized without prompting.
+     * List of account {@link "@wallet-standard/base".WalletAccount.address addresses} that the app is requesting authorization to use.
+     * If provided, the Wallet must return *only* accounts with *any* of the provided addresses.
+     */
+    readonly addresses?: WalletAccount['address'][];
+    /**
+     * List of account {@link "@wallet-standard/base".WalletAccount.chains chains} that the app is requesting authorization to use.
+     * If provided, the Wallet must return *only* accounts with *any* of the provided chains.
+     */
+    readonly chains?: WalletAccount['chains'];
+    /**
+     * List of account {@link "@wallet-standard/base".WalletAccount.features features} that the app is requesting authorization to use.
+     * If provided, the Wallet must return *only* accounts with *any* of the provided features.
+     */
+    readonly features?: WalletAccount['features'];
```

Comment:
Alas, this doesn't let the wallet express feature/version pairs, should that be necessary.

Reply:
Well, the wallet still has versions on features, the accounts don't. That said, the way _version_ is implemented right now (because there's only one per named feature, and you really want more than one) is busted. It might be time to fix that anyway.